great_expectations/data_context/config_validator/yaml_config_validator.py

```diff
@@ -492,12 +415,7 @@ def _test_instantiation_of_data_connector_from_yaml_config(
             config_defaults={},
         )
 
-        anonymizer = Anonymizer(self._data_context.data_context_id)
-
-        usage_stats_event_payload = anonymizer.anonymize(
-            obj=instantiated_class, name=data_connector_name, config=config  # type: ignore[arg-type]
-        )
-        return instantiated_class, usage_stats_event_payload
+        return instantiated_class, {}
```

Comment:
Why the empty dict?

Reply:
We should remove that and chance the return type to just `DataConnector`

Reply:
this entire class will be deleted (we no longer want to support `test_yaml_config`) - I can adjust with a singular return type though

Reply:
Oh, sorry, if it's going away in the very near term, I'm fine with with leaving it as is until it's gone.

Reply:
I've updated anyways!